flake.nix

```diff
@@ -235,6 +236,50 @@
               export PATH="${alejandra.defaultPackage.${system}}/bin"
               ${pkgs.treefmt}/bin/treefmt --clear-cache "$@"
             '');
+
+          docs.type = "app";
+          docs.program =
+            l.toString
+            (pkgs.writeScript "docs" ''
+              # convert to lowercase, all of our doc paths are in lowercase
```

Comment:
Can we move that script into another file? And maybe we should expose this via devshell menu

Reply:
Yeah that sounds alright, I'll do that :+1: